feat: add features block and icons

divyanshugarg36 opened the pull request:
This adds the feature block to the site which is https://www.figma.com/file/nI1IDdPUHRr2vzBNOLIr60/BRL-CAD-Wireframe?node-id=0%3A96 in the Figma design.

divyanshugarg36 committed:
feat: add features block and icons

divyanshugarg36 committed:
feat: add new block about brl-cad

theamanjs commented:
can you add description to the PR?

theamanjs commented on src/components/Features.tsx:
```diff
@@ -0,0 +1,50 @@
+      title: t('FEATURES.FEATURE_3.TITLE'),
+      description: t('FEATURES.FEATURE_3.DESCRIPTION'),
+    },
+  ];
+  return (
+    <section className="features">
```
There are some places where everything is joined line to line. If it's possible and to make code breath a little can we add a new line after the function ends, return starts, prop types start, etc. You can make a decision where to add new line, it will make code a little more cleaner

theamanjs replied:
@divyanshugarg36 just a reminder, not an issue.

divyanshugarg36 replied:
Oh yeah,
This one has been taken care of
Each component have required spacings

But I will do another sweep in the whole project to ensure this

theamanjs reviewed (changes requested):
can you work on resolving the comment I did just now?

divyanshugarg36 committed:
chore: fix spacing issue in whole project for functions and return statements

divyanshugarg36 committed:
Merge branch 'features' of https://github.com/divyanshugarg36/brl-cad-web into what

divyanshugarg36 committed:
feat: add scss flex mixin to make flex usage more efficient

divyanshugarg36 committed:
Merge branch 'flex' of https://github.com/divyanshugarg36/brl-cad-web into highlights

divyanshugarg36 commented:
> can you add description to the PR?

Done

divyanshugarg36 commented:
> can you work on resolving the comment I did just now?

This has been fixed in latest commit of this PR

divyanshugarg36 committed:
feat: add highlight block

- add new images
- update css issues

divyanshugarg36 committed:
chore: update component name from what to information

divyanshugarg36 committed:
Merge branch 'what' of https://github.com/divyanshugarg36/brl-cad-web into highlights

divyanshugarg36 committed:
chore: update brl logo to common component

- fix highlights data

divyanshugarg36 committed:
feat: add about us section

- add new hook for body scroll on modal open
- add modal component
- add new cross icon

divyanshugarg36 committed:
feat: add project list section in home page

divyanshugarg36 committed:
Merge pull request #17 from divyanshugarg36/highlights

feat: add highlight block

divyanshugarg36 committed:
Merge pull request #13 from divyanshugarg36/what

feat: add new block about brl-cad

divyanshugarg36 committed:
refactor: add new project list

- move text to i18n

divyanshugarg36 committed:
feat: add project page using project cards

- add new page title component

divyanshugarg36 committed:
chore: add spacing for common components

divyanshugarg36 committed:
refactor: move about-us object to constants

divyanshugarg36 committed:
Merge pull request #26 from divyanshugarg36/project-page

feat: add project page using project cards

divyanshugarg36 committed:
Merge pull request #21 from divyanshugarg36/project-section

feat: add project list section in home page

divyanshugarg36 committed:
Merge pull request #20 from divyanshugarg36/about-us

feat: add about us section
